template.yaml

```diff
@@ -400,3 +407,21 @@ Resources:
                   - kms:Decrypt
                   - kms:GenerateDataKey
                 Resource: '{{resolve:ssm:SqsKmsKeyArn}}'
+
+  # Query results integration test resources
+
+  IntegrationTestsSqsOperationsFunctionNameParameter:
+    Condition: TestEnvironment
+    Type: AWS::SSM::Parameter
+    Properties:
+      Name: !Sub /tests/${AWS::StackName}/SqsOperationsFunctionName
+      Type: String
+      Value: '{{resolve:ssm:/tests/SqsOperationsFunctionName}}'
+
+  IntegrationTestsWriteTestDataToAthenaBucketQueueUrlParameter:
+    Condition: TestEnvironment
+    Type: AWS::SSM::Parameter
+    Properties:
+      Name: !Sub /tests/${AWS::StackName}/WriteTestDataToAthenaBucketQueueUrl
+      Type: String
+      Value: '{{resolve:ssm:WriteTestDataToAthenaBucketQueueUrl}}'
```

Comment:
This might be in another repo, so ignore me for now if you can't change. But this ssm param should include /tests at the start of the path too.

i.e. `'{{resolve:ssm:/tests/WriteTestDataToAthenaBucketQueueUrl}}'

Reply:
Done.